tools/autograd/templates/Functions.cpp

```diff
@@ -51,7 +52,16 @@ void copy_range(variable_list& out, IndexRange range, at::ArrayRef<Tensor> t) {
 }
 
 std::vector<Tensor> to_tensor_list(const variable_list& variables) {
-  return fmap(variables, [](const Variable &v) { return static_cast<Tensor>(v); } );
+  IntList sizes;
```

Comment:
This doesn't match the name `to_tensor_list` anymore. :) Could you instead have a custom `unbind_backward`?

Reply:
Sure thing! Do you want to keep to_tensor_list?

Reply:
If it's not used elsewhere, feel free to delete it. :)